common/djangoapps/student/views.py

```diff
@@ -739,7 +739,7 @@ def dashboard(request):
     }
 
     ecommerce_service = EcommerceService()
-    if ecommerce_service.is_enabled():
+    if ecommerce_service.is_enabled() and request.user.is_active:
```

Comment:
I'm seeing multiple instances of this combination-check -- maybe we can bundle it up into a useful helper function?  What does it mean if the service is enabled but the user is not active?  What question are we trying to answer?